src/NzbDrone.Core/MediaFiles/EpisodeImport/Aggregation/Aggregators/AggregateSubtitleInfo.cs

```diff
@@ -0,0 +1,59 @@
+using System;
+using System.IO;
+using System.Linq;
+using NLog;
+using NzbDrone.Common.Instrumentation;
+using NzbDrone.Core.Download;
+using NzbDrone.Core.Extras.Subtitles;
+using NzbDrone.Core.Parser;
+using NzbDrone.Core.Parser.Model;
+
+namespace NzbDrone.Core.MediaFiles.EpisodeImport.Aggregation.Aggregators
+{
+    public class AggregateSubtitleInfo : IAggregateLocalEpisode
+    {
+        private static readonly Logger Logger = NzbDroneLogger.GetLogger(typeof(AggregateSubtitleInfo));
+        public int Order => 6;
+
+        public LocalEpisode Aggregate(LocalEpisode localEpisode, DownloadClientItem downloadClientItem)
+        {
+            var path = localEpisode.Path;
+            var isSubtitleFile = SubtitleFileExtensions.Extensions.Contains(Path.GetExtension(path));
+            if (!isSubtitleFile)
```

Comment:
New line before block
```suggestion

            if (!isSubtitleFile)
```